guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+Puppet modules enable specific configurations based on host data gathered by Facter.
+
+[[Facts]]
+Facts:: Host parameters such as total memory, operating system version, or architecture.
+Facts are reported by Facter and used by Puppet.
+
+[[Foreman]]
+Foreman:: The component mainly responsible for provisioning and content lifecycle management.
+ifdef::satellite[]
+Foreman is the main upstream counterpart of {ProjectName}.
+endif::[]
+
+[[{project-context}_Services]]
+{Project} services:: A set of services that {ProjectServer} and {SmartProxyServers} use for operation.
+You can use the `{foreman-maintain}` tool to manage these services.
+To see the full list of services, enter the `{foreman-maintain} service list` command on the machine where {Project} or {SmartProxyServer} is installed.
+
+[[Foreman_Hook]]
+Foreman Hook:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
+endif::[]
+
+[[Full_Host_Image]]
+Full Host Image:: A boot disk used for PXE-less provisioning of a specific host.
+The full host image contains an embedded Linux kernel and init RAM disk of the associated operating system installer.
+
+[[Generic_Image]]
+Generic Image:: A boot disk for PXE-less provisioning that is not tied to a specific host.
+The generic image sends the host’s MAC address to {ProjectServer}, which matches it against the host entry.
+
+[[Hammer]]
+Hammer:: A command line tool for managing {ProjectName}.
+You can execute Hammer commands from the command line or utilize them in scripts.
+Hammer also provides an interactive shell.
+
+[[Host]]
+Host:: Refers to any system, either physical or virtual, that {ProjectName} manages.
+
+[[Host_Collection]]
+Host Collection:: A user defined group of one or more Hosts used for bulk actions such as errata installation.
+
+[[Host_Group]]
+Host Group:: A template for building a host.
+Host groups hold shared parameters, such as subnet or lifecycle environment, that are inherited by host group members.
+Host groups can be nested to create a hierarchical structure.
+
+[[Host_Image]]
+Host Image:: A boot disk used for PXE-less provisioning of a specific host.
+The host image only contains the boot files necessary to access the installation media on {ProjectServer}.
+
+[[Incremental_Update]]
+Incremental Upgrade (of a Content View):: The act of creating a new (minor) content view version in a lifecycle environment.
+Incremental upgrades provide a way to make in-place modification of an already published content view.
+Useful for rapid updates, for example when applying security errata.
+
+[[Job]]
+Job:: A command executed remotely on a host from {ProjectServer}.
+Every job is defined in a job template.
+
+[[Job_Template]]
+Job Template:: Defines properties of a job.
+
```

Comment:
```suggestion
```
I think "Job template" is pretty self-explanatory. It's a template for a job (and the term `job` is explained above).

Reply:
Dropped.